eventstore/tasks.py

```diff
@@ -119,3 +130,125 @@ def delete_rapidpro_contact_by_uuid(contact_uuid):
     | delete_contact_pii.s()
     | delete_rapidpro_contact_by_uuid.s()
 )
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
+    retry_backoff=True,
+    max_retries=1,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def get_rapidpro_contact_by_urn(urn):
+    if not urn:
+        return
+    return rapidpro.get_contacts(urn=urn).first(retry_on_rate_exceed=True).serialize()
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
```

Comment:
We're not making any http requests in this method, so it doesn't make sense to autoretry for HTTP methods. It also probably doesn't make sense to retry with backoff either, since we're not contacting external services. `acks_late` still makes sense though (for lost workers), as well time timeouts.